Update Set-PolicyConfig.md (#11533)
MicrosoftDocs/office-docs-powershell: exchange/exchange-ps/exchange/Set-PolicyConfig.md

Accept wildcard characters: False
```

### -ExtendTeamsDlpToSpoOdbConsent

> This flag is to enable Teams DLP Policy to automatically extend protection to the content stored in OneDrive shared in 1:1/n chats and content stored in SharePoint associated with Teams teams shared through channel chats.

Reviewer: ```suggestion
The ExtendTeamsDlpToSpoOdbConsent parameter enables Teams DLP policy to automatically extend protection to the content stored in OneDrive shared in 1:1 chats and content stored in SharePoint associated with Teams teams shared through channel chats.
```
